### ISSUE TITLE
Subtitles (WebVTT) not added to HLS playlist when running mp4dash with --hls

### ISSUE THREAD

**Issue by nilmic** [axiomatic-systems/Bento4#659]
Got some errors when running mp4dash and the Subtitles (WebVTT) are missing in the master HLS playlist.

How to reproduce:
`Bento4-SDK-1-6-0-639.x86_64-unknown-linux/bin/mp4dash --profiles=live --hls --output-dir=/tmp/mp4dash-errors /tmp/F0110641_806kbps.fmp4 /tmp/F0110641_320kbps.fmp4 /tmp/F0110641_1312kbps.fmp4 /tmp/F0110641_2069kbps.fmp4 /tmp/F0110641_3100kbps.fmp4 [+format=webvtt,+language=swe,+language_name=Svenska]/var/packager-testfiles/F0110641/219714-13.vtt`

Errors:
```
Line 1043: ERROR: local variable 'language_name' referenced before assignment
# Might be fixed by changing line 1043 "language_name" to "subtitles_file.language_name"

Line 1044: ERROR: 'SubtitlesFile' object has no attribute 'hls_autoselect'
# Might be fixed by changing line 1044 "'YES' if subtitles_file.hls_autoselect else 'NO'," to "'YES' if hasattr(subtitles_file, 'hls_autoselect') and subtitles_file.hls_autoselect else 'NO',"
```
Suggested changes as a patch :
```
--- Bento4-SDK-1-6-0-639.x86_64-unknown-linux/utils/mp4-dash.py.orig	2021-11-09 14:04:47.698302013 +0100
+++ Bento4-SDK-1-6-0-639.x86_64-unknown-linux/utils/mp4-dash.py	2021-11-16 10:41:51.057855875 +0100
@@ -1040,8 +1040,8 @@
             if default:
                 default_selected = True
             master_playlist_file.write('#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subtitles",NAME="{}",AUTOSELECT={},DEFAULT={},LANGUAGE="{}",URI="{}/{}"\n'.format(
-                                       language_name,
-                                       'YES' if subtitles_file.hls_autoselect else 'NO',
+                                       subtitles_file.language_name,
+                                       'YES' if hasattr(subtitles_file, 'hls_autoselect') and subtitles_file.hls_autoselect else 'NO',
                                        'YES' if default else 'NO',
                                        subtitles_file.language,
                                        media_subdir,
```

**Comment by barbibulle** [comment]
Thanks for reporting this.
The issue should be fixed with the latest commit, which fixes #667 (same issue)

**Comment by clia6547** [comment]
> date
I run this on version 1.6.0-639 and got ERROR: local variable 'language_name' referenced before assignment
